torch/csrc/distributed/c10d/Ops.cpp

```diff
@@ -97,6 +97,19 @@ c10::intrusive_ptr<ProcessGroup::Work> scatter_(
       ScatterOptions{root_rank, std::chrono::milliseconds(timeout)});
 }
 
+c10::intrusive_ptr<ProcessGroup::Work> alltoall_(
+    at::TensorList output_tensors,
```

Comment:
why the type here is different from other comm ops?

Reply:
not stamping yet due to this comment

Reply:
I think it's a convention to use TensorList to represent const std::vector<at::Tensor>&.

I only use const std::vector<at::Tensor>& if there is a const std::vector<std::vector<at::Tensor>>& in the signature to keep consistency on that function. Let me know which way you like.

Reply:
I see. can we add some docs to explain that in the code? Thanks!